src/main/java/pl/szczesniak/dominik/whattowatch/users/domain/model/UserId.java

```diff
@@ -0,0 +1,21 @@
+package pl.szczesniak.dominik.whattowatch.users.domain.model;
+
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.ToString;
+import pl.szczesniak.dominik.whattowatch.users.domain.model.exceptions.InvalidUserIdValueException;
+
+@EqualsAndHashCode
+@ToString
+public class UserId {
+
+    @Getter
+    private final int value;
+
+    public UserId(final int value) {
+        if (value < 1) {
```

Comment:
zobacz sobie https://www.baeldung.com/guava-preconditions checkArgument, to by było wystarczające, nie trzeba aż tak dużo deklarować swoich wyjątków